### PR TITLE
ARROW-8741: [Python][Packaging] Keep VS2015 with for the windows wheels

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
@github-actions crossbow submit  wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-*

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-*

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-*

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-cp36m

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-*

**Comment by kszucs** [comment]
@github-actions crossbow submit wheel-win-*

File: dev/tasks/python-wheels/appveyor.yml [hunk 1]
```diff
@@ -47,6 +47,9 @@ build_script:
   - rd /s /q C:\OpenSSL-v111-Win32
   - rd /s /q C:\OpenSSL-v111-Win64
 
+  # Disable vcpkg
```
Comment (pitrou) [inline dev/tasks/python-wheels/appveyor.yml:50]: Can you comment why this is necessary?

File: python/pyarrow/tests/conftest.py [hunk 2]
```diff
@@ -199,11 +200,11 @@ def pytest_configure(config):
...
         )
 
-        flag = '--{}'.format(mark)
         enable_flag = '--enable-{}'.format(mark)
+        disable_flag = '--disable-{}'.format(mark)
 
-        is_enabled = (config.getoption(flag) or
-                      config.getoption(enable_flag))
+        is_enabled = (config.getoption(enable_flag) and not
+                      config.getoption(disable_flag))
```
Comment (pitrou) [inline python/pyarrow/tests/conftest.py:207]: Why the complication? `--xxx` is much nicer than `--enable-xxx`. And `--enable-xxx --disable-xxx` doesn't make sense.

**Review by pitrou — CHANGES_REQUESTED** [review]

File: python/pyarrow/tests/conftest.py (hunk 2 above)
Comment (kszucs) [reply python/pyarrow/tests/conftest.py:207]: The absence of `--xxx` means to use the default for that flag which depends on an optional import most of the cases.

Comment (pitrou) [reply python/pyarrow/tests/conftest.py:207]: I see, thanks.

**Comment by kszucs** [comment]
Since:
- I'm preparing the 0.17.1 patch release 
- My last comment addition shouldn't affect the build statuses 
- GHA services has an outage at the moment
I'm going to merge it.

**Review by kszucs — COMMENTED** [review]
+1

**Review by kszucs — COMMENTED** [review]
+1

**Pull request closed without merging**